src/java.base/share/lib/security/default.policy

```diff
@@ -21,6 +21,8 @@ grant codeBase "jrt:/java.net.http" {
     permission java.lang.RuntimePermission "accessClassInPackage.jdk.internal.misc";
     permission java.lang.RuntimePermission "modifyThread";
     permission java.net.SocketPermission "*","connect,resolve";
+    // required if the HTTPClient is configured to use a local bind address
+    permission java.net.SocketPermission "localhost:*","listen,resolve";
```

Comment:
Acked: I see that binding a `SocketChannel` will call `SecurityManager::checkListen`.
I'd have to double check with `SocketPermission` to verify `"localhost:*"` covers all the cases (that is - that it is sufficient to bind on any local interface)